helm/charts/crossplane-provider-btp-cf/values.ci.yaml

```diff
@@ -0,0 +1,106 @@
+---
+########################################################################################################################
+###! /!\ this file is only used in github pipeline to template a valid and NOT empty helm manifest! #####
+########################################################################################################################
+apps:
+  - cloudfoundryBtpOrchestrateCloudProviderConfigRefName: "cloudfoundryBtpOrchestrateCloudProviderConfigRefName"
+    name: "name"
+    forProvider:
+      name: my-app
+      dockerImage: cloud-orchestration-experimental.common.repositories.cloud.sap/hello-cf:latest
```

Comment:
@ricogu please anonymise all internal sap informations/links in all files.

Reply:
done